src/base/bittorrent/session.cpp

```diff
@@ -3044,13 +3049,18 @@ void Session::startUpTorrents()
         QByteArray data;
     } TorrentResumeData;
 
-    auto startupTorrent = [this, logger, resumeDataDir](const TorrentResumeData &params)
+    int addCount = 0;
```

Comment:
Could you replace `addCount` with more sensible and unambiguous name, please? This one is too general. `startedTorrentsCount` or something alike.